core/src/data/value/into.rs

```diff
@@ -637,10 +759,4 @@ mod tests {
             Ok(I::Month(274))
         )
     }
-
-    #[test]
-    fn try_into_u128() {
-        let uuid = 195965723427462096757863453463987888808;
-        assert_eq!((&Value::Uuid(uuid)).try_into() as Result<u128>, Ok(uuid))
-    }
```

Comment:
This codes removal looks not intentional.
It could happen when you add support for unsigned 128-bit int type.

Reply:
Yes, I removed this when I removed the u128 parts..  I'll make sure I put it back in.. Thanks for catching this!